src/button/component.jsx

```diff
@@ -426,10 +427,22 @@ export let Button : Component<ButtonOptions> = create({
             },
             decorate({ allowed = [], disallowed = [] } : Object = {}, props : ButtonOptions) : {} {
 
+                // remove Venmo from our allowed list if the rendering device is not a mobile one
                 if (allowed && allowed.indexOf(FUNDING.VENMO) !== -1 && !isDevice()) {
                     allowed = allowed.filter(source => (source !== FUNDING.VENMO));
                 }
 
+                // Uncookied venmo ramp. Even without 'pwv' cookie, we'll be rendering venmo button
+                // for a sample of the mobile population. 
+                if (allowed && allowed.indexOf(FUNDING.VENMO) === -1 && isDevice()) {
```

Comment:
I would also check if it's in remembered group -- if so should be ineligible for the experiment.

Reply:
ah, good call